src/flapjackfeeder.c

```diff
@@ -90,13 +92,13 @@ int nebmodule_init(int flags, char *args, nebmodule *handle) {
     /* set some info - this is completely optional, as Nagios doesn't do anything with this data */
     neb_set_module_info(npcdmod_module_handle, NEBMODULE_MODINFO_TITLE, "flapjackfeeder");
     neb_set_module_info(npcdmod_module_handle, NEBMODULE_MODINFO_AUTHOR, "Birger Schmidt");
-    neb_set_module_info(npcdmod_module_handle, NEBMODULE_MODINFO_TITLE, "Copyright (c) 2013 Birger Schmidt");
-    neb_set_module_info(npcdmod_module_handle, NEBMODULE_MODINFO_VERSION, "0.0.3");
+    neb_set_module_info(npcdmod_module_handle, NEBMODULE_MODINFO_TITLE, "Copyright (c) 2013-2015 Birger Schmidt");
```

Comment:
That should probably be `NEBMODULE_MODINFO_COPYRIGHT`, otherwise you're re-setting the first info line.